api/controllers/CommunityController.js

```diff
@@ -228,14 +228,18 @@ module.exports = {
   },
 
   removeMember: function (req, res) {
-    Membership.query().where({
+    return Membership.query().where({
       user_id: req.param('userId'),
       community_id: req.param('communityId')
     }).update({
       active: false,
       deactivated_at: new Date(),
       deactivator_id: req.session.userId
     })
+    .then(() => Community.find(req.param('communityId'))
+      .then(community => community.save({
+        num_members: community.get('num_members')
```

Comment:
i think you want a `- 1` here

but then, to skip the 2 queries of "find then save", you should use [decrement](http://knexjs.org/#Builder-decrement)